microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -34,23 +34,31 @@ When fixing a bug, you often want to try out some changes to your JavaScript cod
 
 To view and edit a JavaScript file:
```

Comment:
pls try a rewrite of intro to streamline it, to focus on only the present article, and to clearly differentiate the 3 articles.

eg:
This article covers X aspects of Y.
See also:
Article 2 - covers Z aspects of Y.
Article 3 - covers W aspects of Y.

Probably delete my sentence "The tutorial has screen captures, so you can skim it." - more confusing than helpful.

"You can easily try out the debugger features by using the demo webpage." -- this is confusing: which demo for which article?  Probably need to name and specify which demo page, for which article.  Maybe delete that sentence, which isn't about the present article, and is confusing, excess detail, out-of-place.